### ISSUE TITLE
Qt GUI scale issue

### ISSUE THREAD

**Issue by rgovindjee** [TUDelft-CNS-ATM/bluesky#491]
In both WSL (Ubuntu 22.04) and Windows 10 natively I'm seeing a graphics bug when running with Qt graphics where all waypoints appear as giant triangles (see image). Aircraft also appear at a drastically incorrect scale, and this does not change with zoom level. This does not appear in when using `pygame`, but we're having issues with FF mode in pygame (possibly unrelated?), and would therefore like to use Qt.

![qt_bug](https://github.com/TUDelft-CNS-ATM/bluesky/assets/15008656/1217f495-acb7-47f7-8085-1925333b8891)

**Comment by jooste** [comment]
This has been reported once before, and seems to be related to an incompatibility between a shader setting and your video card driver. I can't reproduce this, so it's difficult to fix. I'll have a look again though, who knows :)

**Comment by jooste** [comment]
Hi @rgovindjee, the only difference with other (working) GL functions that I can see is the use of `QOpenGLShaderProgram.setAttributeValue` to set a fixed scale for screen-coordinates shapes. I've just committed a potential fix which directly uses `glVertexAttrib1f` instead. Could you try and see if this fixes your issue?

**Comment by jooste** [comment]
Yes, although it is not actively maintained, so it could be that incompatibilities exist.

Could you let me know a bit more about the system you are having these issues on? You were saying both windows and linux, so drivers shouldn't be the problem. What video card do you have? It would be useful if we could reproduce this problem ourselves.

**Comment by jooste** [comment]
Ok, in that case it looks like the `switch` statement in the shader is not honoured. Let's see if we can come up with an alternative approach.

**Comment by TejEddineLakhal** [comment]
I came across this graphics bug on Windows 10. In my case setting python.exe to use "high-performance NVIDIA processor" instead of "integrated graphics" as preferred graphics processor in the NVIDIA control panel fixes the issue. My system configuration is: 

- Operating System:  Windows 10 
- CPU: Intel Core i5-5200U
- GPU: Nvidia GeForce 820M

**Comment by jooste** [comment]
Hey @TejEddineLakhal, thanks a lot, that's a useful suggestion!

**Comment by samuel-baldacchino** [comment]
Hi, @TejEddineLakhal's suggestion worked for me as well. Thank you. Just in case you find it useful, my system configuration is:

- Operating System: Windows 10
- CPU: Intel Core i5-8300H
- GPU: Nvidia GeForce GTX 1050
